src/components/VerifyIdentity.js

```diff
@@ -65,14 +65,15 @@ class VerifyIdentity extends Component {
       const vettingOptionsObject = vettingRegistry(!this.props.valid_nin);
       // extract the keys from the vettingOptionsObject
       const vettingOptionsKeys = Object.keys(vettingOptionsObject);
+      const addedNin = this.props.nins[0];
       vettingButtons = [
         <div key="1" id="nins-btn-grid"> 
           {vettingOptionsKeys.map((key, index) => {
             let helpText = buttonHelpTextArray[index];
             return (
               <div key={index}>
                 {vettingOptionsObject[key]}
-                <p key={index} className="proofing-btn-help">
+                <p key={index} className={"proofing-btn-help" + (index < 2 && !addedNin ? " disabled":"")}>
```

Comment:
Could we add a comment here to remind us to check how this works in case we reorder the buttons (and different index values will be assigned to different buttons)?

Reply:
What do you think about the idea of structuring the code in a way that all of this disable functionality can target both the phone and the letter button depending on nin or not and leave freja separate? For me it would improve the readability of the code. Do you have a different plan for the future?

Reply:
Now we have object from vetting-registry that is looping object in verifyIdentity,
If we separate depending on nin we will need to separate vetting-registry for phone/letter and eidas/frejaeID, then both of them need to loop verifyIdentity independent... 
I think keeping the functions together would be better since they are looping the same object